diff --git a/config.h b/config.h
index 0985d2e..763ac48 100644
--- a/config.h
+++ b/config.h
@@ -1,6 +1,9 @@
+#include <X11/XF86keysym.h>
+
 ///---User configurable stuff---///
 ///---Modifiers---///
-#define MOD             XCB_MOD_MASK_4       /* Super/Windows key  or check xmodmap(1) with -pm  defined in /usr/include/xcb/xproto.h */
+#define MOD             XCB_MOD_MASK_1       /* Alt_L  or check xmodmap(1) with -pm  defined in /usr/include/xcb/xproto.h */
+#define MOD3		XCB_MOD_MASK_3       /* Alt_R */
 ///--Speed---///
 /* Move this many pixels when moving or resizing with keyboard unless the window has hints saying otherwise.
  *0)move step slow   1)move step fast
@@ -21,7 +24,8 @@ static const uint8_t offsets[] = {0,0,0,0};
  *6)emptycol         */
 static const char *colors[] = {"#35586c","#333333","#7a8c5c","#ff6666","#cc9933","#0d131a","#000000"};
 /* if this is set to true the inner border and outer borders colors will be swapped */
-static const bool inverted_colors = true;
+static const bool inverted_colors = false;
+static const bool enable_compton = true;
 ///---Cursor---///
 /* default position of the cursor:
  * correct values are:
@@ -40,7 +44,21 @@ static const uint8_t borders[] = {3,5,5,4};
 #define LOOK_INTO "WM_NAME"
 static const char *ignore_names[] = {"bar", "xclock"};
 ///--Menus and Programs---///
-static const char *menucmd[]   = { "", NULL };
+static const char   *menucmd[]   = { "rofi", "-show", "drun", NULL };
+//static const char     *menucmd[]   = { "rofi", "-show", "drun", "-show-icons", NULL };
+static const char     *run_cmd[]   = { "rofi", "-show", "run", "-disable-history", NULL };
+static const char     *termcmd[]   = { "kitty", NULL };
+//static const char     *win_cmd[]   = { "rofi", "-show", "window", "-show-icons", NULL };
+static const char     *win_cmd[]   = { "rofi", "-show", "window",  NULL };
+static const char     *playcmd[]   = { "playerctl", "play-pause", "-p", "spotify,mpd", NULL };
+static const char     *prevcmd[]   = { "playerctl", "previous", "-p", "spotify,mpd", NULL };
+static const char     *nextcmd[]   = { "playerctl", "next", "-p", "spotify,mpd", NULL };
+static const char     *briucmd[]   = { "light", "-A", "5", NULL };
+static const char     *bridcmd[]   = { "light", "-U", "5", NULL };
+static const char     *volucmd[]   = { "pulsemixer", "--change-volume", "+5", NULL };
+static const char     *voldcmd[]   = { "pulsemixer", "--change-volume", "-5", NULL };
+static const char     *voltcmd[]   = { "pulsemixer", "--toggle-mute", NULL};
+
 ///--Custom foo---///
 static void halfandcentered(const Arg *arg)
 {
@@ -93,33 +111,34 @@ static void toggle_sloppy(const Arg *arg)
 #define DESKTOPCHANGE(K,N) \
 {  MOD ,             K,              changeworkspace, {.i=N}}, \
 {  MOD |SHIFT,       K,              sendtoworkspace, {.i=N}},
+#define SHCMD(cmd) {.com=(const char*[]){ "/bin/sh", "-c", cmd, NULL }} // SHCMD macro from dwm
 static key keys[] = {
     /* modifier           key            function           argument */
     // Focus to next/previous window
     {  MOD ,              XK_Tab,        focusnext,         {.i=TWOBWM_FOCUS_NEXT}},
     {  MOD |SHIFT,        XK_Tab,        focusnext,         {.i=TWOBWM_FOCUS_PREVIOUS}},
     // Kill a window
-    {  MOD ,              XK_q,          deletewin,         {}},
+    {  MOD ,              XK_w,          deletewin,         {}},
     // Resize a window
-    {  MOD |SHIFT,        XK_k,          resizestep,        {.i=TWOBWM_RESIZE_UP}},
-    {  MOD |SHIFT,        XK_j,          resizestep,        {.i=TWOBWM_RESIZE_DOWN}},
-    {  MOD |SHIFT,        XK_l,          resizestep,        {.i=TWOBWM_RESIZE_RIGHT}},
-    {  MOD |SHIFT,        XK_h,          resizestep,        {.i=TWOBWM_RESIZE_LEFT}},
+    {  MOD |SHIFT,        XK_k,         resizestep,        {.i=TWOBWM_RESIZE_UP}},
+    {  MOD |SHIFT,        XK_j,         resizestep,        {.i=TWOBWM_RESIZE_DOWN}},
+    {  MOD |SHIFT,        XK_l,         resizestep,        {.i=TWOBWM_RESIZE_RIGHT}},
+    {  MOD |SHIFT,        XK_h,         resizestep,        {.i=TWOBWM_RESIZE_LEFT}},
     // Resize a window slower
-    {  MOD |SHIFT|CONTROL,XK_k,          resizestep,        {.i=TWOBWM_RESIZE_UP_SLOW}},
-    {  MOD |SHIFT|CONTROL,XK_j,          resizestep,        {.i=TWOBWM_RESIZE_DOWN_SLOW}},
-    {  MOD |SHIFT|CONTROL,XK_l,          resizestep,        {.i=TWOBWM_RESIZE_RIGHT_SLOW}},
-    {  MOD |SHIFT|CONTROL,XK_h,          resizestep,        {.i=TWOBWM_RESIZE_LEFT_SLOW}},
+    {  MOD |SHIFT|CONTROL,XK_k,         resizestep,        {.i=TWOBWM_RESIZE_UP_SLOW}},
+    {  MOD |SHIFT|CONTROL,XK_j,         resizestep,        {.i=TWOBWM_RESIZE_DOWN_SLOW}},
+    {  MOD |SHIFT|CONTROL,XK_l,         resizestep,        {.i=TWOBWM_RESIZE_RIGHT_SLOW}},
+    {  MOD |SHIFT|CONTROL,XK_h,         resizestep,        {.i=TWOBWM_RESIZE_LEFT_SLOW}},
     // Move a window
-    {  MOD ,              XK_k,          movestep,          {.i=TWOBWM_MOVE_UP}},
-    {  MOD ,              XK_j,          movestep,          {.i=TWOBWM_MOVE_DOWN}},
-    {  MOD ,              XK_l,          movestep,          {.i=TWOBWM_MOVE_RIGHT}},
-    {  MOD ,              XK_h,          movestep,          {.i=TWOBWM_MOVE_LEFT}},
+    {  MOD ,              XK_k,         movestep,          {.i=TWOBWM_MOVE_UP}},
+    {  MOD ,              XK_j,         movestep,          {.i=TWOBWM_MOVE_DOWN}},
+    {  MOD ,              XK_l,         movestep,          {.i=TWOBWM_MOVE_RIGHT}},
+    {  MOD ,              XK_h,         movestep,          {.i=TWOBWM_MOVE_LEFT}},
     // Move a window slower
-    {  MOD |CONTROL,      XK_k,          movestep,          {.i=TWOBWM_MOVE_UP_SLOW}},
-    {  MOD |CONTROL,      XK_j,          movestep,          {.i=TWOBWM_MOVE_DOWN_SLOW}},
-    {  MOD |CONTROL,      XK_l,          movestep,          {.i=TWOBWM_MOVE_RIGHT_SLOW}},
-    {  MOD |CONTROL,      XK_h,          movestep,          {.i=TWOBWM_MOVE_LEFT_SLOW}},
+    {  MOD |CONTROL,      XK_k,         movestep,          {.i=TWOBWM_MOVE_UP_SLOW}},
+    {  MOD |CONTROL,      XK_j,         movestep,          {.i=TWOBWM_MOVE_DOWN_SLOW}},
+    {  MOD |CONTROL,      XK_l,         movestep,          {.i=TWOBWM_MOVE_RIGHT_SLOW}},
+    {  MOD |CONTROL,      XK_h,         movestep,          {.i=TWOBWM_MOVE_LEFT_SLOW}},
     // Teleport the window to an area of the screen.
     // Center:
     {  MOD ,              XK_g,          teleport,          {.i=TWOBWM_TELEPORT_CENTER}},
@@ -128,12 +147,20 @@ static key keys[] = {
     // Center x:
     {  MOD |CONTROL,      XK_g,          teleport,          {.i=TWOBWM_TELEPORT_CENTER_X}},
     // Top left:
+    {  MOD ,              XK_KP_Home,    teleport,          {.i=TWOBWM_TELEPORT_TOP_LEFT}},
+
     {  MOD ,              XK_y,          teleport,          {.i=TWOBWM_TELEPORT_TOP_LEFT}},
     // Top right:
+    {  MOD ,              XK_KP_Prior,   teleport,          {.i=TWOBWM_TELEPORT_TOP_RIGHT}},
+
     {  MOD ,              XK_u,          teleport,          {.i=TWOBWM_TELEPORT_TOP_RIGHT}},
     // Bottom left:
+    {  MOD ,              XK_KP_End,     teleport,          {.i=TWOBWM_TELEPORT_BOTTOM_LEFT}},
+
     {  MOD ,              XK_b,          teleport,          {.i=TWOBWM_TELEPORT_BOTTOM_LEFT}},
     // Bottom right:
+    {  MOD ,              XK_KP_Next,    teleport,          {.i=TWOBWM_TELEPORT_BOTTOM_RIGHT}},
+
     {  MOD ,              XK_n,          teleport,          {.i=TWOBWM_TELEPORT_BOTTOM_RIGHT}},
     // Resize while keeping the window aspect
     {  MOD ,              XK_Home,       resizestep_aspect, {.i=TWOBWM_RESIZE_KEEP_ASPECT_GROW}},
@@ -167,7 +194,7 @@ static key keys[] = {
     {  MOD ,              XK_comma,      changescreen,      {.i=TWOBWM_NEXT_SCREEN}},
     {  MOD ,              XK_period,     changescreen,      {.i=TWOBWM_PREVIOUS_SCREEN}},
     // Raise or lower a window
-    {  MOD ,              XK_r,          raiseorlower,      {}},
+    {  MOD,              XK_r,          raiseorlower,      {}},
     // Next/Previous workspace
     {  MOD ,              XK_v,          nextworkspace,     {}},
     {  MOD ,              XK_c,          prevworkspace,     {}},
@@ -193,11 +220,25 @@ static key keys[] = {
     {  MOD |SHIFT,        XK_Right,      cursor_move,       {.i=TWOBWM_CURSOR_RIGHT}},
     {  MOD |SHIFT,        XK_Left,       cursor_move,       {.i=TWOBWM_CURSOR_LEFT}},
     // Start programs
-    {  MOD ,              XK_w,          start,             {.com = menucmd}},
+    {  MOD ,              XK_space,                  start, {.com = menucmd}},
+    {  MOD3,              XK_r,                      start, {.com = run_cmd}},
+    {  MOD ,              XK_Return,                 start, {.com = termcmd}},
+    {  0 | CONTROL,       XK_Tab,                    start, {.com = win_cmd}},
+    {  MOD3,              XK_Return,                 start, {.com = playcmd}},
+    {  MOD3,              XK_comma,                  start, {.com = prevcmd}},
+    {  MOD3,              XK_period,                 start, {.com = nextcmd}},
+    {  0   ,              XF86XK_MonBrightnessUp,    start, {.com = briucmd}},
+    {  0   ,              XF86XK_MonBrightnessDown,  start, {.com = bridcmd}},
+    {  0   ,              XF86XK_AudioRaiseVolume,   start, {.com = volucmd}},
+    {  0   ,              XF86XK_AudioLowerVolume,   start, {.com = voldcmd}},
+    {  0   ,              XF86XK_AudioMute,          start, {.com = voltcmd}},
+    {  MOD ,              XK_p,                      start, SHCMD("screenshot full")},
+    {  MOD |CONTROL,      XK_p,                      start, SHCMD("screenshot area")},
+    {  MOD |SHIFT,        XK_p,                      start, SHCMD("screenshot wind")},
     // Exit or restart 2bwm
-    {  MOD |CONTROL,      XK_q,          twobwm_exit,       {.i=0}},
-    {  MOD |CONTROL,      XK_r,          twobwm_restart,    {.i=0}},
-    {  MOD ,              XK_space,      halfandcentered,   {.i=0}},
+    {  MOD |SHIFT,        XK_Escape,           twobwm_exit, {.i=0}},
+    {  MOD |CONTROL,      XK_r,             twobwm_restart, {.i=0}},
+//  {  MOD ,              XK_space,      halfandcentered,   {.i=0}},
     {  MOD ,              XK_s,          toggle_sloppy,     {.com = sloppy_switch_cmd}},
     // Change current workspace
        DESKTOPCHANGE(     XK_1,                             0)
@@ -216,8 +257,8 @@ static Button buttons[] = {
     {  MOD        ,XCB_BUTTON_INDEX_1,     mousemotion,   {.i=TWOBWM_MOVE}, false},
     {  MOD        ,XCB_BUTTON_INDEX_3,     mousemotion,   {.i=TWOBWM_RESIZE}, false},
     {  0          ,XCB_BUTTON_INDEX_3,     start,         {.com = menucmd}, true},
-    {  MOD|SHIFT,  XCB_BUTTON_INDEX_1,     changeworkspace, {.i=0}, false},
-    {  MOD|SHIFT,  XCB_BUTTON_INDEX_3,     changeworkspace, {.i=1}, false},
-    {  MOD|ALT,    XCB_BUTTON_INDEX_1,     changescreen,    {.i=1}, false},
-    {  MOD|ALT,    XCB_BUTTON_INDEX_3,     changescreen,    {.i=0}, false}
+  //{  MOD|SHIFT,  XCB_BUTTON_INDEX_1,     changeworkspace, {.i=0}, false},
+  //{  MOD|SHIFT,  XCB_BUTTON_INDEX_3,     changeworkspace, {.i=1}, false},
+    {  MOD|SHIFT,    XCB_BUTTON_INDEX_1,     changescreen,    {.i=1}, false},
+    {  MOD|SHIFT,    XCB_BUTTON_INDEX_3,     changescreen,    {.i=0}, false}
 };
